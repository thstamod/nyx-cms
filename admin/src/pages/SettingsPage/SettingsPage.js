import React from 'react';
import withMainNavigation from '../../containers/withMainNavigation';


const SettingsPage = () => (
  <div>
    <h1>Settings page</h1>
  </div>
);


export default withMainNavigation(SettingsPage);
