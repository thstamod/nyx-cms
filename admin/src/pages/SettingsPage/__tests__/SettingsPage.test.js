import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import SettingsPage from '../SettingsPage';

test('Settings page Page renders', () => {
  const { asFragment } = render(<SettingsPage />);

  expect(asFragment()).toMatchSnapshot();
});
