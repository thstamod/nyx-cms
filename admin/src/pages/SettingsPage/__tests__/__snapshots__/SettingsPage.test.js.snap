// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Settings page Page renders 1`] = `
<DocumentFragment>
  <div
    class="generalContainer"
  >
    <h1>
      Settings page
    </h1>
  </div>
</DocumentFragment>
`;
